docs/medical-api/more-info/webhooks.mdx

```diff
@@ -128,3 +137,152 @@ The format follows:
 
   </Expandable>
 </ResponseField>
+
+
+### Patient Consolidated Data
+
+A patient's consolidated data is the result of converting C-CDA documents from HIEs into FHIR, as well as other FHIR data your application
+[insert into Metriport's FHIR storage](/medical-api/api-reference/fhir/create-patient-consolidated).
```

Comment:
"... your application has inserted into Metriport's..."

also nit: `patient's` is lowercase here, but in other places in the docs it's capitalized, let's choose one and be consistent